app/views/usages/show.html.erb

```diff
@@ -0,0 +1,17 @@
+<% provide(:title, 'Usage') %>
+<h1>Select Subscriptions</h1>
+<table class='table'>
+  <tr>
+    <th>Subscription Date</th>
+    <th>User</th>
+    <th>Plan</th>
+    <th>Active</th>
+    <th>Actions</th>
+  </tr>
+  <tbody class='tbody'>
+  <% @subscriptions.each { |subscription| %>
+    <%= render 'usages/subscription', subscriptions: @subscriptions %>
+  <% } %>
+  </tbody>
```

Comment:
mentioned earlier as well that this should be done differently. Loop it in the partial rather than on the show page.